[2024-10-20T07:27:01.228+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: crypto_etl.load manual__2024-10-20T07:26:48.683864+00:00 [queued]>
[2024-10-20T07:27:01.239+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: crypto_etl.load manual__2024-10-20T07:26:48.683864+00:00 [queued]>
[2024-10-20T07:27:01.240+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2024-10-20T07:27:01.256+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): load> on 2024-10-20 07:26:48.683864+00:00
[2024-10-20T07:27:01.264+0000] {standard_task_runner.py:57} INFO - Started process 154 to run task
[2024-10-20T07:27:01.267+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'crypto_etl', 'load', 'manual__2024-10-20T07:26:48.683864+00:00', '--job-id', '239', '--raw', '--subdir', 'DAGS_FOLDER/crypto_etl_dag.py', '--cfg-path', '/tmp/tmpn6g4oqxt']
[2024-10-20T07:27:01.269+0000] {standard_task_runner.py:85} INFO - Job 239: Subtask load
[2024-10-20T07:27:01.336+0000] {task_command.py:416} INFO - Running <TaskInstance: crypto_etl.load manual__2024-10-20T07:26:48.683864+00:00 [running]> on host bd1f79790663
[2024-10-20T07:27:01.436+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='crypto_etl' AIRFLOW_CTX_TASK_ID='load' AIRFLOW_CTX_EXECUTION_DATE='2024-10-20T07:26:48.683864+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-20T07:26:48.683864+00:00'
[2024-10-20T07:27:01.717+0000] {logging_mixin.py:154} INFO - Success: Loaded 101 recall records to recalls_db.
[2024-10-20T07:27:01.719+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-10-20T07:27:01.771+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=crypto_etl, task_id=load, execution_date=20241020T072648, start_date=20241020T072701, end_date=20241020T072701
[2024-10-20T07:27:01.842+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-10-20T07:27:01.880+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
